nimble/host/mesh/src/cfg_srv.c

```diff
@@ -735,6 +735,7 @@ static void beacon_set(struct bt_mesh_model *model,
 		BT_ERR("Unable to send Config Beacon Status response");
 	}
 
+fail:
```

Comment:
Let's rename it to "done" as in other places. We should free mbuf irrespective of the result.